multi-node/aws/pkg/config/templates/cloud-config-etcd

```diff
@@ -0,0 +1,16 @@
+#cloud-config
+coreos:
+  update:
+    reboot-strategy: "off"
```

Comment:
I think there should be a cluster-wide reboot strategy, that's why I didn't change this relative to the controller and worker. It could even be a new config option in cluster.yaml.